charts/aws-ebs-csi-driver/templates/node.yaml

```diff
@@ -104,6 +106,10 @@ spec:
           resources:
             {{- toYaml . | nindent 12 }}
           {{- end }}
+          {{- with .Values.node.containerSecurityContext }}
```

Comment:
Looking for feedback: Was wondering if this should default to the `ebs-plugin` security context if not supplied, similar to how resources does (look a few lines up).